src/extractor.rs

```diff
@@ -1006,5 +1012,7 @@ mod tests {
         assert_eq!((res.0).1, "user1");
         assert_eq!((res.1).0, "name");
         assert_eq!((res.1).1, "user1");
+
+        <()>::extract(&req);
```

Comment:
add `.except("To empty extract")`

Reply:
Done

Reply:
It's infallible, so it just returns `()`. I changed it to make that easier to see.